.tekton/operator-bundle-pipeline.yaml

```diff
@@ -105,64 +105,114 @@ spec:
     description: This sets the expiration time for intermediate OCI artifacts produced and used during builds after which they can be garbage collected.
     name: oci-artifact-expires-after
     type: string
+
   - name: operator-image-repo
     description: Image repository for the StackRox Operator.
     type: string
+    default: "quay.io/rhacs-eng/stackrox-operator"
```

Comment:
Please remove `operator-image-repo` value from `.tekton/operator-bundle-build.yaml`.
https://github.com/stackrox/stackrox/blob/0cfad1a5593380c4e1ee8609d59d9e0b8c302253/.tekton/operator-bundle-build.yaml#L34-L35

Reply:
Done in 8543b45408.